code/game/objects/items/weapons/twohanded.dm

```diff
@@ -109,12 +109,17 @@
 	else //Trying to wield it
 		wield(user)
 
+/obj/item/weapon/twohanded/equip_to_best_slot(mob/M)
+	if(..())
+		unwield(M)
+		return
+
 ///////////OFFHAND///////////////
 /obj/item/weapon/twohanded/offhand
 	name = "offhand"
 	icon_state = "offhand"
 	w_class = 5
-	flags = ABSTRACT
+	flags = ABSTRACT|NODROP
```

Comment:
honestly, we should just remove the NODROP augment, that thing fucks up everything